Terminal.Gui/Views/ListView.cs

```diff
@@ -334,7 +334,9 @@ public override void Redraw (Rect bounds)
 			for (int row = 0; row < f.Height; row++, item++) {
 				bool isSelected = item == selected;
 
-				var newcolor = focused ? (isSelected ? ColorScheme.Focus : ColorScheme.Normal) : (isSelected ? ColorScheme.HotNormal : ColorScheme.Normal);
+				var newcolor = focused ? (isSelected ? ColorScheme.Focus
+					: Enabled ? ColorScheme.Normal : ColorScheme.Disabled) : (isSelected ? ColorScheme.HotNormal
+					: Enabled ? ColorScheme.Normal : ColorScheme.Disabled);
```

Comment:
Well, that is effect of the Ctrl+k+D. Currently it uses 3 lines. With if/else it takes 22. What is preferable?

```cs
Attribute newcolor;
if (focused) {
	if (isSelected) {
		newcolor = ColorScheme.Focus;
	} else {
		if (Enabled) {
			newcolor = ColorScheme.Normal;
		} else {
			newcolor = ColorScheme.Disabled;
		}
	}
} else {
	if (isSelected) {
		newcolor = ColorScheme.HotNormal;
	} else {
		if (Enabled) {
			newcolor = ColorScheme.Normal;
		} else {
			newcolor = ColorScheme.Disabled;
		}
	}
}

```

Reply:
I think if we add GetNormalColor method then it would be more readable:
```csharp
var newcolor = focused ? (isSelected ? ColorScheme.Focus : GetNormalColor())
                       : (isSelected ? ColorScheme.HotNormal : GetNormalColor());
```
Where the method would be in `View`:

```csharp
protected Attribute GetNormalColor()
{
    return Enabled ? ColorScheme.Normal : ColorScheme.Disabled;
}
```

Reply:
@tznind thank you very much for your help. Done.